### DISCUSSION TITLE
How to hide API key?

### DISCUSSION THREAD

**Discussion by DavidHTwastaken** [giorgiabosello/google-maps-react-markers#65] [Help]
Hello, I was trying to use google-maps-react for a project and ran into the issue of glitchy markers. This library is awesome for fixing that so thanks very much to the contributors.

My only problem now is that I don't know how to hide the API key. I'm new to React and I'm using Nextjs for the project. Online, it seems like the standard practice is to make API calls on the server-side because everything gets bundled together on the front-end upon building. However, since the API call to Google Maps happens within the code for google-maps-react-markers, how would I go about hiding my API key if I were to make my application available to users?

**Comment by giorgiabosello** [comment]
You don't need to hide it, nobody can use it if it is not enabled in that domain.

**Reply by DavidHTwastaken** [reply]
Ahhhh gotchya thanks so much lolol, that makes sense :)
